src/handlers/web3.js

```diff
@@ -40,22 +53,61 @@ export const web3SetHttpProvider = async network => {
   if (network.startsWith('http://')) {
     web3Provider = new JsonRpcProvider(network, NetworkTypes.mainnet);
   } else {
-    web3Provider = new JsonRpcProvider(
-      replace(infuraUrl, 'network', network),
-      network
-    );
+    web3Provider = new JsonRpcProvider(replace(infuraUrl, 'network', network));
+    return web3Provider.ready;
   }
-  return web3Provider.ready;
```

Comment:
DevSection.js was logging the return value of the ready flag for ganache - was there a reason to move up the `web3Provider.ready` line only for regular networks?